jobs/build/ocp/Jenkinsfile

```diff
@@ -187,285 +242,346 @@ node(TARGET_NODE) {
-                    throw ose_err
+                stage( "puddle: ose 'building'" ) {
+                  buildlib.build_puddle(
+                          PUDDLE_CONF,    // The puddle configuration file to use
+                          PUDDLE_SIGN_KEYS, // openshifthosted key
+                          "-b",   // do not fail if we are missing dependencies
+                          "-d",   // print debug information
+                          "-n",   // do not send an email for this puddle
+                          "-s",   // do not create a "latest" link since this puddle is for building images
+                          "--label=building"   // create a symlink named "building" for the puddle
+                  )
                 }
-                try {
-                    sh "brew watch-task ${OA_TASK_ID}"
-                } catch ( oa_err ) {
-                    echo "Error in openshift-ansible build task: ${OA_BREW_URL}"
-                    throw oa_err
+
+                // TODO: Remove after new OIT version confirmed
+                // Old method for most images
+                stage( "compare dist-git" ) {
+                    sh "ose_images.sh --user ocp-build compare_nodocker --branch rhaos-${BUILD_VERSION}-rhel-7 --group base"
                 }
-            }
 
-            stage( "signing rpms" ) {
-                if ( SIGN_RPMS ) {
-                    sh "${env.WORKSPACE}/build-scripts/sign_rpms.sh rhaos-${BUILD_VERSION}-rhel-7-candidate openshifthosted"
-                } else {
-                    echo "RPM signing has been skipped..."
+                stage( "update dist-git" ) {
+                    sh "ose_images.sh --user ocp-build update_docker --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --force --release '${NEW_DOCKERFILE_RELEASE}' --version 'v${NEW_VERSION}'"
                 }
-            }
+                // End old method
 
-            stage( "puddle: ose 'building'" ) {
-              buildlib.build_puddle(
-                      PUDDLE_CONF,    // The puddle configuration file to use
-                      PUDDLE_SIGN_KEYS, // openshifthosted key
-                      "-b",   // do not fail if we are missing dependencies
-                      "-d",   // print debug information
-                      "-n",   // do not send an email for this puddle
-                      "-s",   // do not create a "latest" link since this puddle is for building images
-                      "--label=building"   // create a symlink named "building" for the puddle
-              )
-            }
+                // OIT Method
+                OIT_WORKING = "${pwd(tmp=true)}"
 
-            stage( "compare dist-git" ) {
-                sh "ose_images.sh --user ocp-build compare_nodocker --branch rhaos-${BUILD_VERSION}-rhel-7 --group base"
-            }
+                stage( "update dist-git" ) {
+                  sh "echo 'ose: ${env.OSE_DIR}' > ${env.WORKSPACE}/sources.yml"
```

Comment:
Can we just use a writeFile?

Reply:
Still learning groovy... I will change that.